cmd/manager/main.go

```diff
@@ -140,6 +142,13 @@ func main() {
 		}
 	}
 
+	log.Info("Creating the TektonInstallation custom resource...")
+	asset, err := tekton.Asset("toolchain.openshift.dev_v1alpha1_tektoninstallation_cr.yaml")
+
+	if err = toolchain.CreateFromYAML(mgr.GetScheme(), mgr.GetClient(), asset); err != nil {
+		log.Error(err, "Failed to create the 'TektonInstallation' custom resource during startup")
+	}
```

Comment:
Shouldn't we wait for the cache being synced? I see that you put this part after starting the controllers, but are you sure that it's not done in separated goroutine after the command is started?

Reply:
good point, I'll fix that!

Reply:
done in https://github.com/codeready-toolchain/toolchain-operator/pull/16/commits/be70053029661a1e4f83bf5db75ff860d43ffc97